### PR TITLE
feat: Make include_file overrule include_ext, fixes: #350, base_on: #358

### PR DESCRIPTION
**Pull request by zhb127**
Fixes: #350 

Base on: #358 

I don't know, why marked #358 PR as draft?

Just fix code by #358 PR author's comments. then create new PR.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Please add "fixes" reference to https://github.com/cosmtrek/air/issues/350 to your comment.

**Comment by zhb127** [comment]
> Please add "Fixes: #350" to your comment.

OK

**Comment by silverwind** [comment]
Bump, would love to see this merged.

**Comment by nmfzone** [comment]
Just to notify, I personally want to see this merged

**Comment by park-jemin** [comment]
Bump - would love to see this merged as well, seems it's been hanging for a while

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,8 @@ LABEL maintainer="Rick Yu <cosmtrek@gmail.com>"
 ENV GOPATH /go
 ENV GO111MODULE on
 
-COPY . /go/src/github.com/cosmtrek/air
-WORKDIR /go/src/github.com/cosmtrek/air
+COPY . /go/src/github.com/zhb127/air
```
Comment (flada-auxv) [inline Dockerfile:8]: @zhb127 Why are these links also being updated ..? I would be happy if this PR was merged, but it looks like these need to be fixed.

Comment (zhb127) [reply Dockerfile]: Sorry, I've restored the code.

Comment (zhb127) [reply Dockerfile]: I originally thought this feature wasn't accepting , so I made modifications for the forked repository.
sorry @flada-auxv

Comment (flada-auxv) [reply Dockerfile]: No, you don't need to apologize. Thank you for your response!

**Comment by silverwind** [comment]
You should also include this change to actually test the changed functionality: https://github.com/cosmtrek/air/pull/358/files#diff-c484509c25b7c8e4df331c9aefcbb49e43881a9a57c201a4491a4b7b61a72acb

**Comment by zhb127** [comment]
> You should also include this change to actually test the changed functionality:
> 
> https://github.com/cosmtrek/air/pull/358/files#diff-c484509c25b7c8e4df331c9aefcbb49e43881a9a57c201a4491a4b7b61a72acb

thx, The test code has been added.

**Pull request merged**